netbox/virtualization/models/virtualmachines.py

```diff
@@ -127,6 +127,11 @@ class VirtualMachine(ContactsMixin, ImageAttachmentsMixin, RenderConfigMixin, Co
         null=True,
         verbose_name=_('disk (GB)')
     )
+    serial_number = models.CharField(
```

Comment:
I just noticed that we've named the field `serial_number` here but it's called `serial` on the Device and Rack models. IMO we should keep the name consistent across all models to avoid confusion. (Apologies for not registering this earlier.)